test/metabase/api/table_test.clj

```diff
@@ -504,3 +501,49 @@
 (expect
   []
   ((user->client :crowberto) :get 200 "table/card__1000/fks"))
+
+(defn narrow-fields [category-names api-response]
```

Comment:
if this isn't being used outside this namespace I would vote that we make it private. That way if we need to tweak it in the future we know it only affects tests in this namespace so we don't need to grep the codebase looking for other usages